app/components/forms/wizard/basic-details-step.js

```diff
@@ -269,6 +269,40 @@ export default Component.extend(FormMixin, EventWizardMixin, {
             }
           ]
         },
+        ticketMinPrice: {
+          identifier : 'min_price',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Minimum price for donation tickets required')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Invalid number')
+            },
+            {
+              type   : 'integer[1..]',
+              prompt : this.l10n.t('Ticket price should be greater than 0')
+            }
+          ]
+        },
+        ticketMaxPrice: {
+          identifier : 'max_price',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Maximum price for donation tickets required')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Invalid number')
+            },
+            {
+              type   : 'integer[1..]',
```

Comment:
the rule be used here is, hat max price should be greater than min price, read sematic ui validation docs.